pkg/transport/setup/visor.go

```diff
@@ -0,0 +1,87 @@
+package setup
+
+import (
+	"context"
+	"fmt"
+	"net/rpc"
+
+	"github.com/skycoin/dmsg"
+	"github.com/skycoin/dmsg/cipher"
+	"github.com/skycoin/skycoin/src/util/logging"
+
+	"github.com/skycoin/skywire/pkg/skyenv"
+	"github.com/skycoin/skywire/pkg/transport"
+	"github.com/skycoin/skywire/pkg/visor/visorconfig"
+)
+
+// TransportListener provides an rpc service over dmsg to perform skycoin transport setup
+type TransportListener struct {
+	dmsgC   *dmsg.Client
+	log     *logging.Logger
+	tm      *transport.Manager
+	tsNodes []cipher.PubKey
+}
+
+// NewTransportListener makes a TransportListener from configuration
+func NewTransportListener(ctx context.Context, conf *visorconfig.V1, dmsgC *dmsg.Client, tm *transport.Manager) (*TransportListener, error) {
+	log := logging.MustGetLogger("transport_setup")
+	go dmsgC.Serve(ctx)
+	log.WithField("local_pk", conf.PK).Info("Connecting to the dmsg network.")
+	select {
+	case <-dmsgC.Ready():
+		log.Info("Connected!")
```

Comment:
Could you please expand this log line a bit? Will it be clear enough in the overall logs flow?